src/people/main/FocusView.tsx

```diff
@@ -263,7 +277,22 @@ function FocusedView(props: FocusViewProps) {
       props?.ReCallBounties();
   }
 
-  let initialValues: any = {};
+  //this workflow now gets the org user is in and appends it
+  const searchKey = uuid;
+  let variableToAppend = '';
+
+  const foundObject = userOrganizations.find((obj: any) => obj.value === searchKey);
```

Comment:
can we rename `foundObject` to something more unique to this scenario seems like pretty generic naming